### PR TITLE
Mobile-ready layout

### LINKED ISSUE #18: Responsive/mobile layout

### PR DESCRIPTION
**Pull request by eturk**
I basically made the layout mobile-ready by hiding the big sidebar on mobile (toggle is in the top navigation). I'd appreciate some testing of it on different devices.

(This would resolve #18)

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -36,6 +36,11 @@
             <li class="nav__item nav__item--header">
               <a class="nav__link nav__link--header" href="http://status.siteleaf.com/">Status</a>
             </li>
+            {% if page.nav_toggle != false %}
+              <li class="nav__item nav__item--header nav__toggle">
+                <span class="nav__link nav__link--header">&#9776;</span>
```
Comment (sawyerh) [inline _layouts/default.html]: This button should probably be a `<button>` element. Besides being more semantically correct, it's more accessible and I _think_ gives you friendlier button interactions on phones, for free.

**Comment by sawyerh** [comment]
This is looking good :+1: A few things:

---

I would update this branch with `master`, because I think you're going to see that this happens:

image

---

At the moment, the `.wrap` and `.container` elements have a width of `85%`, which gives us a decent bit of margin on the sides. We could probably get away with bumping this up to `90%`, which would give us more room for our line length to span when the screen gets smaller.

---

The blocks on the homepage could use a bit more attention. When they collapse to 3 columns, they no longer span the full-width (same with the 1 column layout):

image

image

**Comment by eturk** [comment]
Do we need the "Contribute" and "Status" links on mobile?

**Comment by sawyerh** [comment]
> Do we need the "Contribute" and "Status" links on mobile?

I don't think we can assume that they're not needed, but they probably don't need to be displayed as prominently in the header. We could put it in the list of links that display when clicking the hamburger button.

**Comment by sawyerh** [comment]
>  learn.siteleaf.com jump start instructions are really really great -- it's like a Missing Manual -- but I find it quite difficult to use on mobile devices. IMHO, you should improve the whole page and contextual menu layout using Flexbox.

from an email earlier today. Good time :+1:

**Comment by sawyerh** [comment]
You're probably still working on this, but FYI this mobile menu no longer displays and still need to change the menu toggle button to a `<button>` element :+1: 

menu

**Comment by eturk** [comment]
mobile

I'm trying to make the sidebar nav a little less unwieldy. Does collapsing the lists under the section titles make things a little easier?

**Comment by sawyerh** [comment]
I think we can ship this after a few small changes:

- [ ] Show full lists by default. It makes the nav long, but I think I prefer that over guessing which section I need to expand to look in.
- [ ] Don't hide search. Search is important on mobile and Swiftype has a decent mobile experience.

**Comment by sawyerh** [comment]
Can we ship this?

**Pull request merged**